Makefile

```diff
@@ -225,7 +226,7 @@ $(THRIFT_GEN): $(THRIFT_FILES) $(BIN)/thriftrw $(BIN)/thriftrw-plugin-yarpc | $(
 PROTO_ROOT := proto
 # output location is defined by `option go_package` in the proto files, all must stay in sync with this
 PROTO_OUT := .gen/proto
```

Comment:
is this still relevant?

Reply:
Yes, since we still generate all internal types here. Those that are used for history/matching services.
Only public API types were moved to cadence-idl repo, internal types stays here.